be/src/exec/es/es_predicate.cpp

```diff
@@ -298,8 +302,12 @@ Status EsPredicate::build_disjuncts_list(const Expr* conjunct) {
         } else {
             is_not_null = true;
         }
+        std::string col = slot_desc->col_name();
```

Comment:
Repeat three times.
```
        std::string col = slot_desc->col_name();
        if (_field_context.find(col) != _field_context.end()) {
            col = _field_context[col];
        }
```

Reply:
maybe instroduce a common function would increase the complexity